src/BlazorWasmSdk/Tasks/BootJsonData.cs

```diff
@@ -98,6 +98,14 @@ public class ResourcesData
         /// </summary>
         [DataMember(EmitDefaultValue = false)]
         public Dictionary<string, ResourceHashesByNameDictionary> extensions { get; set; }
+
+        /// <summary>
+        /// Extensions created by users customizing the initialization process. The format of the file(s)
+        /// is up to the user.
```

Comment:
What do you mean by "users"? This isn't public API, is it?

Could we have a more specific message, for example stating that this is used for injecting assets from the runtime?